.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+### Motivation
+- Please give a brief description for the background of this change.
+- Issue number: 
+
+
+### Modifications
+#### Change summary
+Please describe what changes are included in this CR. 
```

Comment:
Can you update CR to code change or change request? Everything else looks good.

Reply:
Changed.